packages/gatsby-remark-copy-linked-files/src/__tests__/index.js

```diff
@@ -303,11 +394,12 @@ describe(`gatsby-remark-copy-linked-files`, () => {
     it(`copies file to the destination supplied by destinationDir`, async () => {
       const markdownAST = remark.parse(`![some absolute image](${imagePath})`)
       const validDestinationDir = `path/to/dir`
+      const fileLocationPart = `some-hash/${imageName}.gif`
       const expectedNewPath = path.posix.join(
         process.cwd(),
         `public`,
         validDestinationDir,
-        `/undefined/undefined.gif`
+        fileLocationPart
```

Comment:
this and other adjustments are not necessary for this PR, but as I looked to add cases for assetPrefix I noticed those `undefined` there in expecteted strings and thought that it doesn't inspire a lot of confidence in those tests so I adjusted tests to specify `internal.contentDigest` and `name` fields on File nodes so those are more specific assertions